src/idpf.rs

```diff
@@ -1785,6 +1785,8 @@ mod tests {
         }
     }
 
+    // XXX We need to update the hard-coded values in this test.
+    #[ignore]
     #[test]
     fn idpf_poplar_public_share_deserialize() {
         let test_vector = load_idpfpoplar_test_vector();
```

Comment:
Per discussion, this can be deleted.

Reply:
Removed the test case and updated the commit message.